### ISSUE TITLE
Qlipper makes Gimp crash when copying image in Gimp

### ISSUE THREAD

**Issue by danyright** [pvanek/qlipper#108]
Hi,
I've been thinking this was a gimp issue for a long time, but just realised it is apparently linked to qlipper.

Steps to reproduce:

1. Start qlipper
2. Start gimp (the issue is also reproducible if you start gimp first)
3. Open an image big enough (my tests indicate an image bigger than around 1500x1500 pixels)
4. Select all and hit CTRL+C to copy the image
5. Gimp crashes

Observations:
- It works flawlessly with smaller images.
- Making a merged copy (Edit > Copy visible) makes it crash with smaller images as a regular copy of the current layer.
- Gimp does not crash if Qlipper is not running.
- Gimp does not crash when running Klipper (KDE's clipboard manager)

System:
- Arch Linux
- Laptop with 8GB RAM, intel i7-7500U
- Running LXQT as desktop environment, with much KDE software

So nothing critical, just thought I'd report it since I figured it out.

Thanks for your work!
D

**Comment by palinek** [comment]
Yes, this behavior is known.... but how can a crash of one app be a problem of different app?

**Comment by dv8472** [comment]
Issue is most probably related to [QTBUG-56595](https://bugreports.qt.io/browse/QTBUG-56595).
Have been experiencing similar crashes [like this](https://forum.palemoon.org/viewtopic.php?f=37&t=24846).

**Comment by lucazade** [comment]
same here.. probably related to store images in clipboard.
[https://gitlab.gnome.org/GNOME/gimp/-/issues/2733](https://gitlab.gnome.org/GNOME/gimp/-/issues/2733)

**Comment by ilopmar** [comment]
Hello! 

In the [Gimp issue](https://gitlab.gnome.org/GNOME/gimp/-/issues/2733) some users that use another clipboard manager mention that disabling copy images in their clipboard manager makes this issue go away. I really love qlipper and I would like to continue using, but it seems there is no option to just ignore keeping images on the clipboard. Do you think this can be done?

**Comment by ilopmar** [comment]
> Yes, this behavior is known.... but how can a crash of one app be a problem of different app?

I don't know the internals but my guess is that qlipper does something with the clipboard when copying a big image on Gimp that makes Gimp fails.

**Comment by baptx** [comment]
Is there a way to ignore images like mentioned by @ilopmar? So we don't have to close Qlipper. On https://www.gimp-forum.net/Thread-Gimp-2-10-crashing-every-time-I-try-to-cut-or-copy, someone said there is an option for another clipboard manager (Klipper) but I did not see it for Qlipper.
